packages/extension/src/chromeServices/background/index.ts

```diff
@@ -426,27 +426,25 @@ chrome.runtime.onMessage.addListener(
         }
       });
     } else if (type === 'REQUEST_SUBMIT_TRANSACTION/V3') {
-      handleTransactionRequest({
-        payload: message.payload,
-        sender,
-        parameter: PARAMETER_SUBMIT_TRANSACTION,
-        receivingMessage: 'RECEIVE_SUBMIT_TRANSACTION/V3',
-        errorPayload: {
-          type: ResponseType.Reject,
-          result: undefined
-        }
-      });
+      const { payload } = message;
+      try {
+        sendInMemoryMessage({
```

Comment:
I think that this function is not properly named.
```suggestion
        sendMessageInMemory({
```